src/api_client.rs

```diff
@@ -4,26 +4,26 @@
 use crate::error::ApiError;
 use crate::types::{
     ApiResult, AttestationDuty, BalanceSummary, BeaconHeaderSummary, BeaconProposerRegistration,
-    BlockId, CommitteeDescriptor, CommitteeFilter, CommitteeSummary, DepositContract, EventTopic,
-    FinalityCheckpoints, GenesisDetails, HealthStatus, PeerSummary, ProposerDuty, PublicKeyOrIndex,
-    RootData, StateId, SyncCommitteeDescriptor, SyncCommitteeDuty, SyncCommitteeSummary,
-    SyncStatus, ValidatorStatus, ValidatorSummary, Value, VersionData,
+    BlockId, CommitteeDescriptor, CommitteeFilter, CommitteeSummary, Coordinate, DepositContract,
+    EventTopic, FinalityCheckpoints, GenesisDetails, HealthStatus, PeerSummary, ProposerDuty,
+    PublicKeyOrIndex, RootData, StateId, SyncCommitteeDescriptor, SyncCommitteeDuty,
+    SyncCommitteeSummary, SyncStatus, ValidatorStatus, ValidatorSummary, Value, VersionData,
 };
 #[cfg(feature = "peer-id")]
 use crate::types::{NetworkIdentity, PeerDescription, PeerDescriptor};
+use ethereum_consensus::altair::mainnet::{
```

Comment:
any reason these were re-imported for altair?

we will want some kind of solution to handle multiple phases/presets/configs but that would come later

Reply:
ah, yea that can be reverted back to `phase0`. I originally thought the missing field might just be a versioning issue.
```suggestion
use ethereum_consensus::phase0::mainnet::{
```